src/Mod/PartDesign/WizardShaft/WizardShaftTable.py

```diff
@@ -40,15 +40,15 @@ class WizardShaftTable:
     }
     rowDictReverse = {}
     headers = [
-                "Length [mm]",
-               "Diameter [mm]",
-               "Inner diameter [mm]",
-               "Constraint type",
-               "Start edge type",
-               "Start edge size",
-               "End edge type",
-               "End edge size"
-              ]    
+               QT_TRANSLATE_NOOP("WizardShaftTable", "Length [mm]"),
```

Comment:
For this and all other lines:
```suggestion
               QtCore.QT_TRANSLATE_NOOP("WizardShaftTable", "Length [mm]"),
```